src/Microsoft.Framework.PackageManager/Sources/ListSourcesCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+
+namespace Microsoft.Framework.PackageManager
+{
+    internal class ListSourcesCommand
+    {
+        public Reports Reports { get; set; }
```

Comment:
This seems to be required and if it null it will fail. Why don't you pass it as a constructor arg ?